src/lib/reasoners/satml_frontend.ml

```diff
@@ -60,6 +60,7 @@ module Make (Th : Theory.S) : Sat_solver_sig.S = struct
     skolems : E.gformula ME.t; (* key <-> f *)
     add_inst : E.t -> bool;
     guards : guards;
+    ur : Sat_solver_sig.unknown_reason option;
```

Comment:
I am not very fond of very short names but in this case the type makes clear what is the purpose of the field.

Reply:
I agree with you on this one, I'm changing it (and adding some documentation)